definitions/fenix.toml

```
@@ -596,6 +596,22 @@ description = "Events Ping"
 build_id_column = "REPLACE(CAST(DATE(mozfun.norm.fenix_build_to_datetime(client_info.app_build)) AS STRING), '-', '')"
 
 
+[data_sources.events_stream]
+from_expression = """(
+    SELECT
+        p.*,
+        DATE(p.submission_timestamp) AS submission_date
```

Not a big deal, but it's better to select what you need if possible. I think you should be ok with just this:
```suggestion
        legacy_telemetry_client_id AS client_id,
        profile_group_id,
        experiments,
        event_category,
        event_name,
        event_extra,
        DATE(p.submission_timestamp) AS submission_date
```
but up to you.